app/views/wpcc/your_details/new.html.erb

```diff
@@ -45,9 +45,6 @@
           <%= f.form_row :gender do %>
             <div class="details__field-label">
               <%= f.label :gender, t('wpcc.details.gender.label'), class: 'form__label-heading' %>
-              <%= popup_tip_trigger options: {
-                text: t('wpcc.tooltip_show')
-              } %>
```

Comment:
When we remove the popup component can we also remove the `data-dough-component="PopupTip"` attribute from the parent? (Means that the component doesn't initialise on load)